activerecord/lib/active_record/connection_adapters/sqlite3/quoting.rb

```diff
@@ -0,0 +1,36 @@
+module ActiveRecord
+  module ConnectionAdapters
+    module SQLite3
+      module Quoting # :nodoc:
+        def quote_string(s)
+          @connection.class.quote(s)
+        end
+
+        def quote_table_name_for_assignment(table, attr)
+          quote_column_name(attr)
+        end
+
+        def quote_column_name(name)
+          @quoted_column_names[name] ||= %Q("#{name.to_s.gsub('"', '""')}")
+        end
+
+        private
+
+        def _type_cast(value)
```

Comment:
This method inherits from [lib/active_record/connection_adapters/abstract/quoting.rb#L176-L189](https://github.com/rails/rails/blob/v5.0.0.beta2/activerecord/lib/active_record/connection_adapters/abstract/quoting.rb#L176-L189).